public/app/core/components/Login/LoginForm.tsx

```diff
@@ -1,4 +1,5 @@
-import React, { PureComponent, SyntheticEvent, ChangeEvent } from 'react';
+import React, { ChangeEvent, PureComponent, SyntheticEvent } from 'react';
+import { Selectors } from '@grafana/e2e/src/selectors';
```

Comment:
Using the same `Selectors` from in the real app.  Will need some help in future PRs from @dprokop as I got errors importing from `e2e` root dir because of `Cypress` types.

Reply:
In the meantime, can you drop what's the error? Are you exporting cypress types too?

Reply:
The problem is I can't expose Cypress types to the Grafana app because the would overwrite all `jest` types, I think

Reply:
I get for instance errors that `toEqual`, `toBe` can't be found